lib/config.js

```diff
@@ -9,6 +9,9 @@ module.exports = (options) => {
 		name: packageJson.name,
 		version: packageJson.version,
 		baseUrl: "/",
+		host: null,
+		basePath: null,
+		schemes: null,
```

Comment:
ik persoonlijk zou het niet onlogisch vinden mochten zoveel mogelijk props een sensible default krijgen,  zelfs al is schemes optioneel ik vind dat toch proper dat het uiteindelijk in die swagger json staat, zodat als je een API moet implementeren je alle info hebt om hem aan te roepen.
nu kan je daar op 2 manieren naar kijken: moet dat per se als default staan of vereis je dat van de swagger docs dat het uiteindelijk door de developer is ingevuld... 
zeker dat 2e, maar als alles wat we tegenwoordig bouwen op https beschikbaar is kan je dat evengoed als 'default' value gebruiken neen?
wat denk jij @ThomasBormans